app/export.go

```diff
@@ -348,47 +356,41 @@ func (a *App) ExportAllPosts(writer io.Writer) *model.AppError {
 
 			postLine := ImportLineForPost(post)
 
-			// Do the Replies.
-			replies, err := a.buildPostReplies(post.Id)
+			postLine.Post.Replies, err = a.buildPostReplies(post.Id)
 			if err != nil {
 				return err
 			}
 
-			reactions, err := a.BuildPostReactions(post.Id)
-			if err != nil {
-				return err
+			postLine.Post.Reactions = &[]ReactionImportData{}
```

Comment:
Not sure if I'm missing something obvious, but why do you need this line?

Reply:
I set this here to make sure that it wasn't nil in case `HasReactions` is false.

I did this as a precaution (I wasn't sure if it was okay to return a nil value or if something could potentially reference the reactions value assuming it wasn't nil)

Reply:
OK, I doubt it matters but no substantive objection.